### PR TITLE
[python] Don't set PYTHONCASEOK=1.

### PR DESCRIPTION
**Pull request by ghost**
Comment from SVN says it "should really be removed" and this fixes import
errors like:
  https://github.com/trakt/script.trakt/issues/190#issuecomment-70185992

This affected non-Windows + non-Android (= OS X), but it really doesn't make any sense
considering that the vast majority of add-ons do not depend on the platform
they're running on (so someone trying to import "a" in place of "A" on Linux
would get an ImportError, but it'd work on OS X).

tl;dr it's only causing random obscure errors.
## 

See https://docs.python.org/3.2/using/cmdline.html#envvar-PYTHONCASEOK
@tamland perhaps?

### CODE REVIEW

**Comment by topfs2** [comment]
+1. The point of scripts over binary addons is to be cross platform. Its
not always possible but limiting these accidental platform stuff seems very
very sane.

If it turns out an addon need it, for a good reason, highly doubt one
exists though. We could enable it if add-on.xml  contains a flag and if
platforms are appropriately set.

Nice find BTW!
Den 16 jan 2015 18:22 skrev "menakite" <notifications@github.com>:

**Comment by tamland** [comment]
I have no idea but that does sound very weird..
Maybe @Memphiz knows (os x specific)

**Comment by Memphiz** [comment]
#if !defined(TARGET_WINDOWS) && !defined(TARGET_ANDROID) is not only osx - its osx/ios/linux

@davilla any idea? I can't think of anything why this should be needed for any platform tbh.

**Comment by Memphiz** [comment]
Maybe this has something to do with case sensitive filesystems? no clue

**Comment by tamland** [comment]
@Memphiz The docs above say "[PYTHONCASEOK] only works on Windows, OS X, and OS/2." Hence osx/ios only.

**Comment by Memphiz** [comment]
Now its getting even stranger - on osx and all platforms which use unified depends we are using python 2.6.5:

https://docs.python.org/2.6/using/cmdline.html#envvar-PYTHONCASEOK

Not quiet sure because the docs are for 2.6 but this suggests in python 2.6.x it only works on windows anyways ... ^^

**Comment by tamland** [comment]
@Memphiz It doesn't say it was changed in 2.7, which all such changes should. I can only assume it was documentation error fixed in 2.7

If no objections I see no point keeping old dumb hack no one can remember what was for:)
